### PR TITLE
Updated missing forwarder search query to use the latest event detail

### PR DESCRIPTION
**Pull request by mahirchavda**
Changes:
* Updated missing forwarder search query to use the latest event detail

### CODE REVIEW

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 1]
```diff
@@ -1938,9 +1938,9 @@ display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
 search = (index=_internal sourcetype=splunkd (connectionType=cooked OR connectionType=cookedSSL) fwdType=* group=tcpin_connections) \
-| stats first(sourceIp) as sourceIp, first(fwdType) as forwarder_type, latest(version) as version, latest(arch) as arch, latest(os) as os, max(_time) as last_connected by hostname \
+| stats first(sourceIp) as sourceIp, first(fwdType) as forwarder_type, first(version) as version, first(arch) as arch, first(os) as os, max(_time) as last_connected by hostname \
 | inputlookup append=true cs_forwarders.csv \
-| stats last(sourceIp) as sourceIp, last(forwarder_type) as forwarder_type, last(version) as version, last(arch) as arch, last(os) as os, max(last_connected) as last_connected by hostname \
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1943]: For the first search, we can keep the latest for the first line as that is directly from the events.
Second-line looks okay.

**Review by VatsalJagani — CHANGES_REQUESTED** [review]

File: cyences_app_for_splunk/default/savedsearches.conf (hunk 1 above)
Comment (mahirchavda) [reply cyences_app_for_splunk/default/savedsearches.conf:1943]: First stats command uses both **first** and **latest**. Let me know if you are fine with changing all **first** to **latest** for first stats command

Comment (VatsalJagani) [reply cyences_app_for_splunk/default/savedsearches.conf:1943]: yeah lets change all to latest for first line.

**Review by VatsalJagani — CHANGES_REQUESTED** [review]

File: cyences_app_for_splunk/default/savedsearches.conf (hunk 1 above)
Comment (mahirchavda) [reply cyences_app_for_splunk/default/savedsearches.conf:1943]: Update PR

**Review by VatsalJagani — APPROVED** [review]

**Pull request merged**